templates/minio/statefulset.yaml

```diff
@@ -81,8 +97,20 @@ spec:
           successThreshold: 1
           failureThreshold: 3
         volumeMounts:
+        - mountPath: /tmp
```

Comment:
Could I suggest that we place these extra dirs on the existing PVC?

we whould change to this pattern:
```yaml
        volumeMounts:
        - mountPath: /bitnami/minio/data
          name: liferay-minio-pvc
          subPath: data
        - mountPath: /opt/bitnami/minio/tmp
          name: liferay-minio-pvc
          subPath: mtmp
        - mountPath: /tmp
          name: liferay-minio-pvc
          subPath: tmp
        - mountPath: /.mc
          name: liferay-minio-pvc
          subPath: mc
```
and then delete the `empty-dir` volume.

Make sense?

Reply:
Sure, makes sense, I'll give it a try.